ToDoApp/AddItemViewController/AddItemViewController.swift

```diff
@@ -34,37 +45,51 @@ class AddItemViewController: UIViewController {
         configureUI()
     }
     
-    func getItemName() -> String? {
-        return textField.text
-    }
     
     func configure(addItemViewModel: AddItemViewModel) {
         self.addItemViewModel = addItemViewModel
     }
     
     @objc func goToListVC() {
-        addItemViewModel?.taskManager.addValue(value: textField.text ?? "")
+        addItemViewModel?.getTaskManager().addValue(value: textField.text ?? "")
```

Comment:
let's create function at viewModel addValue(value: String) and call it from viewController, viewController shouldn't know about taskManager